pattern-library/html/boilerplate-header+footer+content+nav.html

```diff
@@ -0,0 +1,113 @@
+        <!--[if lt IE 9]>
+            <p class="alert alert-browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+
+        <div class="nav-skip sr-only">
+            <a class="action-skip" href="#nav-app">Skip to the main navigation</a>
+            <a class="action-skip" href="#content-main">Skip to the main content</a>
+            <a class="action-skip" href="#content-aside">Skip to the aside content</a>
+        </div>
+
+        <!-- wrapper for all UI rendered within the page/view -->
+        <div class="wrapper-view">
+
+            <!-- NOTE: wrapper elements are only needed when they need to manage the grid or larger view/page layout aspects (e.g. spanning the whole viewport) -->
+            <div class="wrapper-header-app">
+                <header class="header-app" role="banner">
+                    <!-- example: app logo/title -->
+                    <h1 class="header-app-title">
+                        <a class="logo" href="{app or company home url}">
+                            <img class="logo-img" src="{path/to/logo-img.svg}" alt="{app/site name}" />
+                            <!-- example: alternate app/site name (if not a graphical logo) -->
+                            <!-- {app/site name} -->
+                        </a>
+                    </h1>
+                    <!-- insert header content here -->
+
+                    <div class="wrapper-navigation">
+                        <nav class="nav-account" id="nav-account">
+                            <!-- NOTE: simplified user account nav example. This would be more complex in reality (with dropdowns, etc.) -->
+                            <a href="{url}">
+                                {username}
+                                <span class="sr-only">'s settings/profile</span>
+                            </a>
+                        </nav>
+                    </div>
+                </header>
+            </div>
+
+            <div class="wrapper-navigation">
+                <nav class="nav-app" id="nav-app">
+                    <a href="{url}">Course Home</a>
+                    <a href="{url}">Course Content</a>
+                    <a href="{url}">Discussions</a>
+                    <a href="{url}">Assignments</a>
+                    <a href="{url}">Assets &amp; Utilities</a>
+                    <a href="{url}">My Progress</a>
+                </nav>
+            </div>
+
+            <!-- NOTE: wrapper elements are only needed when they need to manage the grid or larger view/page layout aspects (e.g. spanning the whole viewport) -->
+            <div class="wrapper-content">
+                <main class="content-main" id="content-main">
+                    <!-- insert main content here -->
+                </main>
+
+                <aside class="content-aside" id="content-aside">
```

Comment:
This would need an aria-label again.  HArder to put a place holder in here because we don't know what the content in the aside should be.  I also caution against include asides in templates.  There is a lot of confusion around what an aside is.  Many think it is analogous to sidebar, but that is not true.  asides should make sense all by themselves, without the rest of the page content as context.  Many times, what you find in asides, is actually more appropriate for ```<footer>``` elements (the author/bio, the date it was published etc) or simply another nested ```<section>```